.semaphore/semaphore.yml

```diff
@@ -25,6 +25,7 @@ global_job_config:
     commands:
       - checkout
       - sem-version java 8
+      - sem-version python unknown
```

Comment:
same here, not sure if adding `sem-version python unknown` will cause any issues?

Reply:
@jainruchir could you remove this line.. we can add it later.. for promotion, we don't need this anyway

Reply:
yeah removed, master also does not set it